src/taxonomy/taxonomy-detail/TaxonomyDetailPage.scss

```diff
@@ -0,0 +1,6 @@
+.taxonomy-detail-page {
+  .sub-header-title-actions {
+    margin-top: 8px;
+    justify-content: flex-start;
```

Comment:
Since there is a preference for usage of classnames for styling where possible, I think this can be replaced with:

`justify-content: flex-start` -> `.justify-content-start`

Reply:
That class is on the SubHeader component. I moved this styles to [SubHeader.scss](https://github.com/openedx/frontend-app-course-authoring/pull/723/commits/d2f5c65a054ec4b4aa295f8b107204a63d945184#diff-0b45c900b531b657541b53815c896132211a4eb63383d016f12d1c0d6f38a2a9R20)

Reply:
Oh sorry I wasn't more clear earlier, I meant the style itself can be added as a class rather than in the scss file, like so: https://getbootstrap.com/docs/4.0/utilities/flex/#justify-content